control/controlclient/map.go

```diff
@@ -310,6 +310,9 @@ func undeltaPeers(mapRes *tailcfg.MapResponse, prev []*tailcfg.Node) {
 				if v := ec.Capabilities; v != nil {
 					n.Capabilities = *v
 				}
+				if v := ec.KeySignature; v != nil {
+					n.KeySignature = v
+				}
```

Comment:
oh, you did make it work.

okay, just bump capver then and put [capver NN] in the commit message?

Reply:
and you can probably probably the deephash mention in the commit message. That's not the main point of the change.

Reply:
Done.